.ci/scripts/boot_simulator.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+set -e
+
+# The name here must match create_simulator.sh
+readonly DEVICE_NAME=Flutter-iPhone
+
+xcrun simctl boot "$DEVICE_NAME" || :
```

Comment:
This could use a comment (i.e., is the `|| :` because this returns non-zero if it's already booted?)